vm/bc/src/MurmurHash3.c

```diff
@@ -28,7 +28,17 @@
 
 #else	// defined(_MSC_VER)
 
-#define	FORCE_INLINE __attribute__((always_inline))
+// RoboVM note: Start change.
```

Comment:
I thought I had replied, but it looks like I didn't!  I just found out where to see this discussion - I had seen these comments in my "app" email but had to use google-fu to find them here.

This is needed because the code does not compile without -fPIC enabled on linux.  But I see you've merged this now, thanks!